backend/pkg/backend/backend.go

```diff
@@ -57,17 +58,17 @@ type Backend struct {
 	monitor             *k8smonitor.Monitor
 	apiInventoryLock    sync.RWMutex
 	dbHandler           _database.Database
-	modules             modules.Module
+	modulesWrapper      modules.Module
```

Comment:
a bit confused, modules.Module isn't the interface for a single module implementation? why is a modules wrapper?

Reply:
The module and the modulesWrapper share the same interface. backend calls functions on the wrapper which in turn dispatches them to each of the modules.

Reply:
I see that the same interface can be used, but i think that the code is not very readable this way, because when we go to the interface description, it's says that it's an interface for a specific module. Maybe we shouldn't abstract it and call the wrapper struct directly (maybe worth to rename it to moduleManager or something), or create a separate interface if we need mocks for testing. WDYT?

Reply:
I'll create another interface that refers to this one, so it will be clear